### PR TITLE
Add What’s New component

### PR DESCRIPTION
**Pull request by GermainBergeron**
Add What’s New component
Rework layout
Tested in IE10, Firefox and Chrome

### CODE REVIEW

File: docs/_components/components/01_header.html [hunk 1]
```diff
@@ -4,8 +4,6 @@
 usage: Use as your web application header. Note that the 9px gap at the bottom of the header is normal and should be hidden by the application container.
 stylesheet: '/scss/components/header.scss'
 ---
-<!--
-Commented because header is position fixed and it's annoying to render it in the Styleguide.
```
Comment (JPLachance) [inline docs/_components/components/01_header.html]: Awesome! :)

File: docs/_includes/header.html [hunk 2]
```diff
@@ -34,7 +34,8 @@
     </div>
 </header>
 
-<nav class="navigation fixed">
+<div class="flex flex-row application-wrapper">
```
Comment (JPLachance) [inline docs/_includes/header.html]: It's kind off weird to open a div in a file that is closed in an other file. It should be opened and closed in the default.html file. Maybe the navigation should be moved out of the header.html file in order for my suggestion to work.

File: docs/_sass/base/_base.scss [hunk 3]
```diff
@@ -13,6 +13,9 @@ dl, dd, ol, ul, figure {
  */
 body {
   background-color: $background-color;
+  flex-flow: column nowrap;
```
Comment (JPLachance) [inline docs/_sass/base/_base.scss]: Woot!

**Comment by JPLachance** [comment]
Really nice, especially the part were our application layout is now handled by flex! Fix the header/footer issue than ship it ;)

File: docs/_components/components/07_whatsnew.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+type: component
+title: What's New
+usage: Use it to provide info on What's New in your application.
+stylesheet: '/scss/components/whats-new.scss'
+---
+<div class='whats-new flex center-align bg-orange'>
```
Comment (wfortin) [inline docs/_components/components/07_whatsnew.html]: `.whats-new` is maybe to precise for a component name. I'd go with something more generic, like `.banner`, `notify-banner`, etc.

File: docs/_components/components/07_whatsnew.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="whats-new-description">
```
Comment (wfortin) [inline docs/_components/components/07_whatsnew.html]: Same here.

File: docs/_layouts/default.html [hunk 6]
```diff
@@ -7,12 +7,16 @@
 
 {% include header.html %}
 
-<div class="page-content application-container">
-    <div class="wrapper">
-        {{ content }}
-    </div>
+<div class="flex flex-row application-wrapper">
```
Comment (wfortin) [inline docs/_layouts/default.html]: :+1:

**Comment by wfortin** [comment]
Let's agree on the naming of the component and then :ship: it.

**Pull request merged**